### PR TITLE
Added my game made specifically for AP, ChecksFinder (Minesweeper)

### PR DESCRIPTION
**Pull request by jonloveslegos**
Contains a client, no options in the yaml, other than the default options, because there are nothing to have as options.
This is the [github](https://github.com/jonloveslegos/ChecksFinder) for the game, it has releases too.
The client does not use a generated file, it just communicates with the server for what it needs to know.
You win the game if you have gotten all your items, and then beat a board after.
The items are, Map Width, Map Height, Map Bombs. Map width and height increase the size of the board in the direction their name states. Map bombs adds a new bomb to the board, but the board is always limited in how many bombs can spawn, but once the board gets bigger that limit will increase.
You unlock checks by getting items, each item gets you a new check, and there are 5 checks that start available to get.
If there is a problem like an item is not sent properly, that can be fixed by closing and reopening both the game and the client.
The client requires you to input the name of the slot when you connect to a server.

### CODE REVIEW

**Comment by JarnoWesthof** [comment]
You might want to also include documentation such as a setup guide, and add the game to the ReadMe

**Comment by jonloveslegos** [comment]
> You might want to also include documentation such as a setup guide, and add the game to the ReadMe

Those are now added.

**Comment by Berserker66** [comment]
Looks like the game is not reporting a goal completion.

File: WebHostLib/static/assets/gameInfo/en_ChecksFinder.md
Comment (Ijwu) [inline WebHostLib/static/assets/gameInfo/en_ChecksFinder.md:10]: Copy paste of MC? Remove this section if not necessary to have.

File: WebHostLib/static/assets/tutorial/checksfinder/checksfinder_en.md
Comment (Ijwu) [inline WebHostLib/static/assets/tutorial/checksfinder/checksfinder_en.md:6]:
```suggestion
  the [Github releases Page for the game](https://github.com/jonloveslegos/ChecksFinder/releases) (latest version)
```

Small nitpick, but probably those reading this guide don't know who 'my' is, even if it's obvious by visiting the github repo.

File: WebHostLib/static/assets/tutorial/checksfinder/checksfinder_en.md
Comment (Ijwu) [inline WebHostLib/static/assets/tutorial/checksfinder/checksfinder_en.md:21]: Empty section, remove if not needed.

File: worlds/checksfinder/Regions.py
Comment (Ijwu) [inline worlds/checksfinder/Regions.py:48]: 'Overworld' seems to be copied from MC. Maybe rename to something more appropriate or exclude the region.

**Review by Ijwu — CHANGES_REQUESTED** [review]

File: worlds/checksfinder/Options.py
Comment (Berserker66) [inline worlds/checksfinder/Options.py:29]: This should be able to be removed now.

**Comment by Ijwu** [comment]
Waiting on approval from @Berserker66 to push this forward.

File: ChecksFinderClient.py
Comment (Berserker66) [inline ChecksFinderClient.py:1]: This entire file seems to be a modified copy of CommonClient.py
Copied code should be avoided where possible, please use imports. See FactorioClient or FF1Client as examples.

File: worlds/checksfinder/Regions.py
Comment (Berserker66) [inline worlds/checksfinder/Regions.py:2]: This looks fairly overkill, as Entrance Randomization doesn't looks to be needed or supported.

File: worlds/checksfinder/__init__.py
Comment (Berserker66) [inline worlds/checksfinder/__init__.py:80]: As it stands this gives the method a docstring with deactived code and makes Main.py spawn a thread for this method as it isn't the default one. Since it looks like no output is needed, I suggest just removing this method.

File: worlds/checksfinder/__init__.py
Comment (Berserker66) [inline worlds/checksfinder/__init__.py:101]: Where does this find use?

**Review by Berserker66 — CHANGES_REQUESTED** [review]

File: ChecksFinderClient.py
Comment (jonloveslegos) [reply ChecksFinderClient.py:1]: I had to keep the server_loop clone because it was overwriting my custom process_server_cmd if I import it for some reason

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**
